Makefile.am

```diff
@@ -287,6 +287,7 @@ clean-local:
 man: $(dist_man_MANS)
 
 .rst:
+	mkdir -p $$(dirname $@)
```

Comment:
Good idea, but should be
        mkdir -p "$(dir $@)"
to let make do the calculation,